documentation/src/main/asciidoc/query.asciidoc

```diff
@@ -339,12 +368,12 @@ Query luceneQuery = mythQB
 ----
 ====
 
-The above query returns results containing "storm" and "tree" either in "name", "history" or "description".
+The above query returns results containing `storm` and `tree` either in `name`, `history` or `description`.
 
-The syntax supported by the SimpleQueryParser includes the following operators: boolean (AND, OR, NOT),
+The syntax supported by the `SimpleQueryParser` includes the following operators: boolean (AND `+`, OR `|`, NOT `-`),
```

Comment:
It's definitely better, I'm a bit worried that users will understand that `AND`, `OR`, etc. can be used as operators in queries, while in fact they cannot.
Maybe something like "(AND using `+`, OR using `|`, NOT using `-`)" would avoid any risk of confusion?